components/Aside.js

```diff
@@ -17,6 +17,7 @@ const FileList = ({ title, files, packageName }) => html`
         <li key=${x.url} data-test=${title + 'Item'}>
           ${FileIcon}
           <a
+            href=${x.url.replace('https://unpkg.com/', '/?')}
             onClick=${e => {
               e.preventDefault();
               pushState(`?${x.url.replace('https://unpkg.com/', '')}`);
```

Comment:
You shouldn’t need an onClick now that you are using the href.

Reply:
I thought about doing this, but using the link instead (i.e. `href` without `preventDefault()`) means that the entire UI is re-rendered, along with all of the scripts, meaning that the page flashes and navigation is less fluid

Reply:
Then perhaps this should be a button and not a link?

Reply:
Good point!

Reply:
Not sure if you'd want a button here-- the behavior you're looking for is normal SPA link handling, so you can use an `a` with a real `href` which works regardless of JS behavior, and then add the client-side routing on top of that. That way you can still right click and open these links in a new tab, for example.

Not as important, but would also be nice to ignore the JS behavior if a modifier key is pressed so that you can open links in new tabs and windows without affecting the current tab: https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/modules/Link.js#L7-L9

Reply:
Thanks for the input! I'll add that logic to the PR

Reply:
This should be an `a` tag with a real `href` like @alexlande suggests. But granted, this is a rudimentary implementation of a link still.

If you want to make a component out of it @VirtualDOMinic then something like this is usually my go to https://github.com/jorgebucaran/hyperapp-router/blob/master/src/Link.js as it covers cases like modifier keys and right clicks.

Reply:
Addressed in commit `d1b37e5`